lib/views/pages/nav_bottom_container/nav_bottom_container.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_ecommerce/views/pages/nav_bottom_container/widgets/nav_bottom_items.dart';
+import 'package:flutter_ecommerce/views/pages/nav_bottom_container/widgets/nav_bottom_screens.dart';
+import 'package:persistent_bottom_nav_bar/persistent-tab-view.dart';
+
+class NavigationBottomContainer extends StatefulWidget {
```

Comment:
This is a duplicated widget.
We can do this refactor on the origin widget and separate the other things in separated widget if possible